Module not found: 'typescript-eslint-parser'

I am trying to import prettier in a create-react-app to format some code snippets, (not trying to using it **on** the project but *in* the project) but I am getting the following error:

```javascript
Error in ./~/prettier/src/parser.js
Module not found: 'typescript-eslint-parser' in /Users/jimmyhmiller/Documents/Code/PlayGround/debug-prettier/node_modules/prettier/src

 @ ./~/prettier/src/parser.js 58:19-54
```

I've put up an [example project](https://github.com/jimmyhmiller/PlayGround/tree/master/debug-prettier). The only change is to `yarn add prettier` and to import it.

I'm guessing that this is because of [the decision](https://github.com/prettier/prettier/pull/915#issuecomment-285168443) to move `typescript-eslint-parser` to devDependencies.

--- a/src/parser.js
+++ b/src/parser.js
@@ -55,16 +55,19 @@ function parseWithBabylon(text) {
 }
 
 function parseWithTypeScript(text) {
-    const parser = require('typescript-eslint-parser')
-    return parser.parse(text, {
-        loc: true,
-        range: true,
-        tokens: true,
-        attachComment: true,
-        ecmaFeatures: {
-            jsx: true,
-        }
-    })
+  // While we are working on typescript, we are putting it in devDependencies
+  // so it shouldn't be picked up by static analysis
+  const r = require;
+  const parser = r('typescript-eslint-parser')
+  return parser.parse(text, {
+    loc: true,
+    range: true,
+    tokens: true,
+    attachComment: true,
+    ecmaFeatures: {
+      jsx: true,
+    }
+  })
 }
 
 module.exports = { parseWithFlow, parseWithBabylon, parseWithTypeScript };

